lib/shared/bucketing-assembly-script/assembly/types/dvcUser.ts

```diff
@@ -299,6 +302,8 @@ export class DVCPopulatedUser extends JSON.Value implements DVCUserInterface {
         json.set('sdkType', this.sdkType)
         json.set('sdkVersion', this.sdkVersion)
         if (this.hostname) json.set('hostname', this.hostname)
+        if (this.clientUUID) json.set('clientUUID', this.clientUUID)
```

Comment:
why are we tying this into the populated user? It's basically unusable as a bucketing value?

Reply:
populated user is more for reporting rather than bucketing. My thought is that it would be useful to align the events to our `sdkConfig` event using this rather than `hostname`

Reply:
it is most definitely for bucketing. The fields it automatically sets are used for platform and platform version filters etc